src/test/java/jalau/at18/katas/Team2/Test2048/BoardTest.java

```diff
@@ -0,0 +1,172 @@
+package jalau.at18.katas.Team2.Test2048;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import jalau.at18.katas.game2048.team2.Board;
```

Comment:
Instead of using import, set file package as `jalau.at18.katas.game2048.team2`

Reply:
We set the correct package